src/index.html

```diff
@@ -7,12 +7,56 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__image"></div>
+      <h3 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h3>
+      <p class="card__text">Product code: 195434</p>
+      <div class="card__item card__item--stars">
+        <ul class="stars">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+        <p class="card__item card__item--text">Reviews: 5</p>
+      </div>
+      <div class="card__item card__item--price">
+        <p class="card__item card__item--price--text">Price:</p>
+        <p class="card__item card__item--price--number">$2,199</p>
+      </div>
+      <button
+        data-qa="hover"
+        type="button"
+        aria-label="Buy now"
+        class="card__btn"
+      >
+        <p class="card__btn card__btn--text">BUY</p>
```

Comment:
In this case, better  use default case and then, styling text to upper/lower/capitalize or other cases

Reply:
Done